systemtests/singlehost/regression_test.go

```diff
@@ -51,7 +51,7 @@ func TestOneHostMultipleNets_regress(t *testing.T) {
 	defer func() {
 		utils.DockerCleanup(t, node1, "myContainer4")
 	}()
-	ipAddress = utils.GetIPAddress(t, node1, "purple-myContainer4")
+	ipAddress = utils.GetIPAddress(t, node1, "purple-myContainer4", "ectd")
```

Comment:
ah, good catch.

I should be using constants as this is error prone, let me take care of this.